21_pattern_examples.md

```
@@ -256,6 +256,161 @@ int main() {
 
 ---
 
+
+/*
+ * Hollow Square Pattern
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 1

(MD007, ul-indent)